infra/openshift-manifests/admin-service/ksvc.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     app: admin-service
 spec:
   template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/min-scale: "1"
```

Comment:
I hated waiting for the admin UI to show up after 10 seconds. Bad for the demo.

It is nicer that we would actually have a single pod always running.

Same story for user UI service.